src/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+.input-search {
+  display: block;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  background: url(images/Search.svg) no-repeat left;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.input-search-big {
+  height: 70px;
+  padding-left: 62px;
+  background-position: left 25px top 25px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input-search-small {
+  height: 42px;
+  padding: 10px 10px 10px 33px;
+  background: url(images/Search.svg) no-repeat left;
+  background-position: left 12px bottom 15px;
+  background-size: 11px 11px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input-search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-search:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: url(images/Search.svg) no-repeat left, linear-gradient(#fff 0%, #f6f6f7 100%);
+  background-position: 20px 20px, 0 0;
+  font-weight: bold;
+  outline: none;
+}
+
+.input-search-small:focus {
+  background: url(images/Search.svg) no-repeat left, linear-gradient(#fff 0%, #f6f6f7 100%);
```

Comment:
There is no need to use `background` rule here because you have the same for `input-search:focus` already.